h2o-r/h2o-package/R/zzz.R

```diff
@@ -48,6 +48,10 @@
   registerS3method("print", "H2OExplanation", "print.H2OExplanation")
   .s3_register("repr", "repr_text", "H2OExplanation")
   .s3_register("repr", "repr_html", "H2OExplanation")
+  
+  # CoxPH
+  .s3_register("survival", "survfit", "H2OCoxPHModel")
```

Comment:
Registering a class with S3 method in one package to a generic from another package is hard in R (unless it's some of the packages that are loaded by default (`base`, `stats` etc) - for those we can use the `registerS3Method`).